.gitlab/pipelines/stages/publish.sh

```diff
@@ -3,5 +3,7 @@
 set -o errexit -o nounset
 
 mc config host add "s3" "https://s3.amazonaws.com" "${AWS_ACCESS_KEY}" "${AWS_SECRET_KEY}"
-mc cp output/kubecf-*.tgz "s3/scf-v3"
-echo "URL: https://scf-v3.s3.amazonaws.com/$(basename output/kubecf-*.tgz)"
+for file in output/kubecf-*.tgz ; do
```

Comment:
This is an unnecessary change. `kubecf-*` should match only one file under `output`. If we get more than one, something is wrong.

Reply:
Ah, yeah, I had made the bundle be called `kubecf-bundle-blah`, but then had to change it to `bundle-blah` because actually deploying it got annoying. Will fix this.

Reply:
Fixed in cb180aa56f8c159344d7888cb62c3976dae57d0d